server/src/reactruntime.js

```diff
@@ -36,9 +36,8 @@ exports.HeadTemplate = class HeadTemplate extends React.Component {
     for (const locale of this.props.userLocales) {
       localeScripts.push(<script key={`l10n-${locale}`} src={this.props.staticLink(`/static/locales/${locale}.js`)} />);
     }
+    const wantsAuth = !this.props.authenticated;
```

Comment:
@chenba Including you this updated wantsAuth include criteria. With this change we will be loading wantsAuth on server only when deviceId doesn't exist

Reply:
We also don't want `wantsAuth` when the user is already authenticated through FxA.